packages/react-components/react-slider/library/src/components/Slider/useSliderStyles.styles.ts

```diff
@@ -135,33 +145,16 @@ const useRailStyles = makeStyles({
     '::before': {
       content: "''",
       position: 'absolute',
-      // Repeating gradient represents the steps if provided
-      backgroundImage: `repeating-linear-gradient(
-        var(${sliderDirectionVar}),
-        #0000 0%,
-        #0000 calc(var(${sliderStepsPercentVar}) - 1px),
-        ${tokens.colorNeutralBackground1} calc(var(${sliderStepsPercentVar}) - 1px),
-        ${tokens.colorNeutralBackground1} var(${sliderStepsPercentVar})
-      )`,
-      // force steps to use HighlightText for high contrast mode
```

Comment:
wondering how is this PR handling these high contrast mode styles when they are removed ?

Reply:
We use the `sliderStepsColorVar` css var for steps color in the gradient, and it's defined a few lines above this one for both default and high contrast mode.